Documentation/ceph-upgrade.md

```diff
@@ -312,3 +333,17 @@ kubectl -n $ROOK_NAMESPACE exec -it $TOOLS_POD -- ceph osd require-osd-release n
 
 #### 4. Verify the updated cluster
 Verify the Ceph cluster's health using the [health verification section](#health-verification).
+
+If you see a health warning about enabling msgr2, please see the section above on [Updating the Mon Ports](#6-update-the-mon-ports).
+```
+[root@minikube /]# ceph -s
+  cluster:
+    id:     b02807da-986a-40b0-ab7a-fa57582b1e4f
+    health: HEALTH_WARN
+            3 monitors have not enabled msgr2
+```
+
+Alternatively, this warning can suppressed if a temporary workaround is needed.
+```
+<insert command>
```

Comment:
@liewegas what is this command to disable the health message? I'm not seeing it in the docs.